### DISCUSSION TITLE
Debian 12 has been released, will there be packages?

### DISCUSSION THREAD

**Discussion by noneofthegoodnames** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#71] [General]
Over a month and a half after the release of Debian 12 aka Bookworm as the new Stable Release and there is still no Waterfox package for it.
Meanwhile there have been updated packages for Debian 11 so it appears this is not abandoned.

Debian Testing now means Debian 13 and the Waterfox package for it from your OBS repo has a dependency which the Debian 12 repos cannot supply.

**Comment by hawkeye116477** [comment]
Thanks for information. I added Debian 12 repo on OBS now and packages started building.

https://build.opensuse.org/package/live_build_log/home:hawkeye116477:waterfox/waterfox-g-kpe/Debian_12/x86_64

**Comment by noneofthegoodnames** [comment]
Thank you, It is much appreciated.
